am-lib/src/main/resources/db/migration/V9__Add_Access_Management_Type_To_Roles_Table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TYPE ACCESSMANAGEMENTTYPE AS enum ('ROLE_BASED', 'EXPLICIT');
```

Comment:
Can we please use underscores to separate words e.g. `ACCESS_MANAGEMENT_TYPE`. We could also rename that one to be `ACCESS_TYPE` to be in sync with Java code base.

I aim to rename `SECURITYCLASSIFICATION` into `SECURITY_CLASSIFICATION` at some point as well ;)

Reply:
You're in luck. SECURITYCLASSIFICATION is now SECURITY_CLASSIFICATION